test/feature/create_group_test.rb

```diff
@@ -45,306 +47,417 @@ class CreateGroupTest < FeatureTest
         page.find("input[placeholder='Name*'][required='required']").wont_be_nil
       end
 
+      it "has button for creating with facebook signup" do
+        page.must_have_button "Create with Facebook"
+      end
+
       it "has button for creating with email signup" do
         page.must_have_button "Create with Email"
       end
 
-      describe "creating subgroup via email signup" do
-        before do
-          fill_out_form group_values_by_input
-          click_button "Create with Email"
-        end
-
-        it "redirects to signup form" do
-          current_path.must_equal "/groups/#{group.id}/subgroups/signup"
-        end
-
-        describe "viewing signup form" do
-          it "has inputs for creating an organizer" do
-            ['Email*',
-             'Password*',
-             'First Name*',
-             'Last Name*',
-             'Zip Code*',
-             'Phone'].each do |label|
-              page.find("input[placeholder='#{label}']").wont_be_nil
-            end
+      describe "creating subgroup" do
+        describe "as a first-time use" do
+          before do
+            OmniAuth.config.mock_auth[:facebook] = mock_facebook_auth
           end
 
-          it "has correct required fields" do
-            ['Email*',
-             'Password*',
-             'First Name*',
-             'Last Name*',
-             'Zip Code*'].each do |label|
-              "input[placeholder='#{label}'][required='required']"
+          describe "via facebook signup" do
```

Comment:
just noting: there is no browser test for google oauth first-time user signup.

i understand that the facebook cases cover almost all of the same logic. however, given how hair the whole thing is (and thus how likely to break for reasons that might elude us later), i'd argue that at least one path for the happy path on the google oauth path might be warranted? (i *think* i did this for the join group tests?) what do you think?

Reply:
and realizing now: no tests to cover facebook (or google) signup for already-existing user (who is or isn't missing contact info)